mylar/webserve.py

```diff
@@ -7815,6 +7818,17 @@ def testslack(self, webhook_url):
             logger.warn('Test variables used [WEBHOOK_URL: %s][USERNAME: %s]' % (webhook_url, username))
             return "Error sending test message to Slack"
     testslack.exposed = True
+    
+    def testmattermost(self, webhook_url):
+        mattermost = notifiers.MATTERMOST(test_webhook_url=webhook_url)
+        result = mattermost.test_notify()
+
+        if result == True:
+            return "Successfully sent Mattermost test -  check to make sure it worked"
+        else:
+            logger.warn('Test variables used [WEBHOOK_URL: %s][USERNAME: %s]' % (webhook_url, username))
+            return "Error sending test message to Slack"
```

Comment:
I don't think you meant to refer to Slack here. Might want to fix this.

Reply:
Whoop, missed that one. Fixed in 28ce225549331e5342ca4ca542c010dbe214a7f1